src/ng/q.js

```diff
@@ -271,7 +271,11 @@ function qFactory(nextTick, exceptionHandler, trackPromiseHandler, untrackPromis
 
   function Promise(keepTrack) {
     this.$$state = { status: 0 };
-    this.keepTrack = keepTrack !== undefined ? keepTrack : true;
+
+    // some built-in angular module may use promises when the dependencies are not yet loaded, make sure not to track those
+    if(trackPromiseHandler && untrackPromiseHandler) {
```

Comment:
Found a bug in Teams app with this new version where some angular module (ng-translate) would use promises without having the qFactory fully initialized (the qFactory is used directly within the module instead of getting it by the angular provider `$QProvider`).